{"uuid": "82b068c9-989a-4d1b-a79b-014fc0688027", "children": ["0f84ad01-15d6-40cf-9069-702c69a89f57"], "befores": [{"name": "_xunit_setup_module_fixture_app_test", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1121, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 893, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\python.py\", line 563, in xunit_setup_module_fixture\n    _call_with_optional_argument(setup_module, request.module)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\python.py\", line 778, in _call_with_optional_argument\n    func()\n  File \"C:\\Users\\TPP07026\\Desktop\\SSO_Project\\app_test.py\", line 44, in setup\n    driver = webdriver.Remote('http://127.0.0.1:4723/wd/hub', desired_caps)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 230, in __init__\n    super().__init__(\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 270, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 320, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 426, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 344, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 366, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\request.py\", line 78, in request\n    return self.request_encode_body(\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\request.py\", line 170, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\poolmanager.py\", line 376, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 703, in urlopen\n    httplib_response = self._make_request(\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 449, in _make_request\n    six.raise_from(e, None)\n  File \"<string>\", line 3, in raise_from\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 444, in _make_request\n    httplib_response = conn.getresponse()\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\http\\client.py\", line 1374, in getresponse\n    response.begin()\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\http\\client.py\", line 318, in begin\n    version, status, reason = self._read_status()\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\http\\client.py\", line 279, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"C:\\Users\\TPP07026\\AppData\\Local\\Programs\\Python\\Python310\\lib\\socket.py\", line 705, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1669948508244, "stop": 1669948519892}], "start": 1669948508244, "stop": 1669948520717}